db/migrate/20160629122413_add_database_name_to_endpoints.rb

```diff
@@ -0,0 +1,5 @@
+class AddDatabaseNameToEndpoints < ActiveRecord::Migration[5.0]
+  def change
+    add_column :endpoints, :database_name, :string
```

Comment:
> I want the URL field to :sparkles: go away :sparkles:

ok. good point. 
so in this case I'd opt for storing the database_name in the path, as this is where it's essentially is, if we treat the endpoint as a URI like `postgres://USER:PASSWORD@HOST:PORT/NAME`

offtopic to this PR: If I understand this endpoint business correct, I'd refactor the db fields to match [URN fragments](https://en.wikipedia.org/wiki/Uniform_Resource_Identifier#Syntax) and, yes, remove `url` and also `ipaddress` (in favor of `hostname`)